static/elements/chromedash-banner.js

```diff
@@ -0,0 +1,59 @@
+import {LitElement, css, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import SHARED_STYLES from '../css/shared.css';
+
+
+class ChromedashBanner extends LitElement {
+  static get properties() {
+    return {
+      timestamp: {type: Number},
+      message: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+    this.timestamp = (new Date()).getTime() / 1000;
```

Comment:
Do we need this initial value? (since there's already `if (!timestamp) return nothing`)

Reply:
Good point.  I set it to null.